src/tests/system/tests/test_ldap.py

```diff
@@ -599,3 +599,391 @@ def test_ldap__reset_cached_timestamps_to_reflect_changes(client: Client, ldap:
     res = client.tools.getent.group("group1")
     assert res is not None, "Group should still exist"
     assert "user1" not in res.members, "User should be removed from group"
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_ldap__nss_override_homedir(client: Client, ldap: LDAP):
```

Comment:
This test case already exists so this can be dropped. 

https://github.com/SSSD/sssd/blob/cf562deb022a7d4d469b7d43c83b86d9bb7e4362/src/tests/system/tests/test_authentication.py#L243C5-L243C67